docs/Release_notes.md

```diff
@@ -1,6 +1,28 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version 3.3.0, released TBD
+--------------------------------------------------------
+
+**Summary:**
```

Comment:
In general I think you should update the title to be more readable and grammar/spelling corrected.

Reply:
What do you have in mind? I was just following the same format as the previous releases